src/api/todos.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable quote-props */
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+type PostDataType = {
+  title:string,
+  completed:boolean
+};
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodo = (userId: number, postData: PostDataType) => {
+  return client.post<Todo>('/todos', {
+    'title': postData.title,
```

Comment:
Use consistent object property quote style. You can remove the quotes from the property names to follow standard convention.